awx/main/migrations/0104_v370_cleanup_old_scan_jts.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.8 on 2020-01-15 20:01
+
+from django.db import migrations, models
+
+
+def cleanup_scan_jts(apps, schema_editor):
+    JobTemplate = apps.get_model('main', 'JobTemplate')
+    JobTemplate.objects.filter(job_type='scan').update(job_type='run')
```

Comment:
this does return the number of records affected, and it might be nice to log some information to the user